arch/risc-v/Kconfig

```diff
@@ -45,6 +45,12 @@ config ARCH_CHIP_ESP32C3
 	select RV32IM_HW_MULDIV
 	select ARCH_VECNOTIRQ
 	select ARCH_HAVE_RESET
+	select LIBC_ARCH_MEMCPY
```

Comment:
`CONFIG_RISCV_MEMCPY` already defines a RISC-V-specific memcpy version. This will probably end up in a double definition error.

Reply:
OK, I will remove it here. OK, I will remove it here. But I add this not for using RISC-V arch level `memcpy`, I mean to use `memcpy` of ESP32C3 ROM, so that the function is not linked to SPI Flash, we can use it without enable I-cache.